libraries/Microsoft.Bot.StreamingExtensions/ReceiveRequestExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// Helper methods added to the <see cref="ReceiveRequest"/> class.
+    /// </summary>
+    public static class ReceiveRequestExtensions
+    {
```

Comment:
Why have these as extension methods? Just put them as methods on the ReceiveRequest class. This file can go away.